SHIR/setup.ps1

```diff
@@ -87,10 +87,10 @@ function RegisterNewNode {
 if (Check-Is-Registered) {
     Write-Log "Restart the existing node"
 
-    if ($ENABLE_HA -eq "true") {
+    if ((Test-Path Env:ENABLE_HA) -and ($Env:ENABLE_HA -eq "true")) {
```

Comment:
Missing single `$` here

Reply:
`Test-Path` is looking for the string which defines the path name much like in [Using the Environment provider and Item cmdlets](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.3#using-the-environment-provider-and-item-cmdlets).

Based on the following commands:
```pwsh
> $Env:APPDATA
C:\Users\User\AppData\Roaming

> Test-Path Env:APPDATA
True
```
you can see that `Test-Path Env:ENABLE_HA` would work as expected. If I run:
```pwsh
$Env:$APPDATA
```
The result is:

> ParserError:
Line |
1 |  $Env:$APPDATA
|  ~~~~~
| Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

So I'm not sure where the `$` sign it's missing.